physionet-django/console/forms.py

```diff
@@ -51,10 +51,10 @@
 
 YES_NO_NA_UNDETERMINED = (
```

Comment:
Instead of having this variable being repeated, it might be better to reference the model (e.g. `CredentialReview.YES_NO_NA_UNDETERMINED.`) to ensure that they stay in sync?

Reply:
Thanks for pointing that out, I removed the variable from the file so that the fields reference the model